app/views/record/_sidebar.html.erb

```diff
@@ -1,4 +1,4 @@
-<div class="col1q-r sidebar">
+<aside class="col1q-r sidebar">
```

Comment:
@JPrevost Yeah, that's fair. It's not part of the main content, imo -- that's the metadata of the record -- but I agree that it's important and directly related.

I landed on `aside` when reviewing the [MDN documentation for `complementary`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/complementary_role), which is the corresponding `role`:

> The complementary [landmark role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles#3._landmark_roles) is used to designate a supporting section that relates to the main content, yet can stand alone when separated.

I could see making it a `div` with accessible markup, but that leads to the question: what do we call this sidebar? It contains a few disparate elements: the download link, the 'ask us' link, and (at least for now) the fact panels. To complicate things further, the download links only appear in GDT.

What do you think about this, or something similar?

`<div role="region" aria-label="Additional information about this record">`

We could also make it `<div role="complementary">`, but I believe that's functionally the same as `<aside>`.

Reply:
I think my concern goes beyond the markup, but the markup helped me to solidify my concern.

Essentially, I think it is the wrong (longterm) choice to put the primary action button "outside" the record like this. I think moving the primary action inside the main element would make it much clearer. It's currently in the "extra stuff I don't care about/it's probably an ad" location on the page rather than "this is clearly where to click if I want this thing".

Setting that aside for now (as it is post MVP for sure to rebuild (or not) that after careful discussion with UX).

```
<div role="region" aria-label="Additional information about this record">
```
Still conveys "this is extra stuff, not the main action" area to me. I think it's on the right track though and maybe we could work with UX to get wording that conveys the "this is the action area" better (and then we might consider adding that as a heading and not just an aria role as well).

Maybe

```
<div role="region" aria-label="Access link and additional information">
```

I'm aware that the link isn't "always" to access the thing, but maybe my wording will help trigger better options that convey that "this is important please check this content".

Reply:
I'm very resistant to including "Access link" in the aria-label for a region that will only include an access link in GeoData. We can use the feature flag to toggle that description, but then the label would be `"Additional information"` for non-GDT, which doesn't feel useful.

With the understanding that we may end up moving that link _outside_ of the sidebar, I would suggest two possible approaches:

1. Leave it as an `aside` for now, which is technically accurate if not perfect.
2. Leave it as an aside for non-GDT apps, but for GDT, change the markup to your suggestion: `<div role="region" aria-label="Access link and additional information">`

Thoughts?

Reply:
Thank you for putting this in context for non-GDT. I was only keeping GDT in mind and agree that when both contexts are included it gets even more complicated.

My preference is your option 2, as I feel strongly that the primary action button should not be in an `<aside>` tag. I don't love having to treat these different but I think we've done enough custom GDT things that one more probably isn't awful. <sigh>